app/scripts/reducers/rules.js

```diff
@@ -59,8 +59,9 @@ export default function reducer (state = initialState, action) {
   const { data, id } = action;
   switch (action.type) {
     case RULE:
+      const foundRule = data.results.find((result) => result.name === id);
```

Comment:
Why was this change necessary?

Reply:
For some reason it was returning results[0] which was the original rule, not the one I just created. so the data filling the newRules page was from the wrong rule.

This even after I updated the fake-api only return the specified rule. but data.results had all rules in there so taking the first one wasn't necessarily correct